modules/ROOT/pages/rest-sdk/simple-composition.adoc

```diff
@@ -0,0 +1,80 @@
+= Simple Composition
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+Simple composition can be used to build a DSL through declarative metadata.
+
+== Simple Composition Patterns
```

Comment:
We're documenting patterns that are not yet supported with the release. v.0.2 will only have support for hiding operations and renaming. cc @jonas19